### ISSUE TITLE
vectorclass.h

### ISSUE THREAD

**Issue by kobe-S9** [p4lang/p4app-switchML#41]
hi,I can't find the code of "vectorclass.h"

**Comment by fruffy** [comment]
Can you provide more detail? Where does this error happen?

**Comment by kobe-S9** [comment]
Due to without Tofino ,I tried to run SAINA（ https://github.com/hotephen/SAINA） which was implemented on top of SwitchML on bmv2. Then I tried to Compile the client_lib Library，the error happened.

**Comment by fruffy** [comment]
Maybe the vectorclass submodule is not correctly installed. It is listed as a submodule here:
https://github.com/p4lang/p4app-switchML/blob/b5c74a04fa66faa82acab154a4e21c8f76d913f2/.gitmodules#L1C1-L3C47


`git submodule init --recursive` try this in the repository.

**Comment by kobe-S9** [comment]
thanks!

**Comment by fruffy** [comment]
Is your problem solved?

**Comment by kobe-S9** [comment]
> Is your problem solved?

no,our goal is run switchML or ATP on bmv2. Due to time constraints，we have to find another way.But I really appreciate your help
